Jetcaster/wear/src/main/java/com/example/jetcaster/MainActivity.kt

```diff
@@ -25,35 +25,51 @@ import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.platform.LocalSavedStateRegistryOwner
+import androidx.compose.ui.res.stringResource
 import androidx.core.splashscreen.SplashScreen.Companion.installSplashScreen
 import androidx.lifecycle.viewmodel.compose.viewModel
+import androidx.navigation.NavHostController
 import androidx.wear.compose.foundation.ExperimentalWearFoundationApi
+import androidx.wear.compose.material.TimeText
+import androidx.wear.compose.navigation.composable
 import androidx.wear.compose.navigation.rememberSwipeDismissableNavController
 import androidx.wear.compose.navigation.rememberSwipeDismissableNavHostState
 import com.example.jetcaster.theme.WearAppTheme
+import com.example.jetcaster.ui.JetcasterNavController.navigateToLatestEpisode
+import com.example.jetcaster.ui.JetcasterNavController.navigateToUpNext
+import com.example.jetcaster.ui.JetcasterNavController.navigateToYourPodcast
+import com.example.jetcaster.ui.LatestEpisodes
+import com.example.jetcaster.ui.home.HomeScreen
+import com.example.jetcaster.ui.home.HomeViewModel
+import com.example.jetcaster.ui.library.LatestEpisodeViewModel
+import com.example.jetcaster.ui.library.LatestEpisodes
 import com.example.jetcaster.ui.player.PlayerScreen
 import com.example.jetcaster.ui.player.PlayerViewModel
 import com.google.android.horologist.annotations.ExperimentalHorologistApi
 import com.google.android.horologist.audio.ui.VolumeViewModel
+import com.google.android.horologist.compose.layout.ScreenScaffold
+import com.google.android.horologist.compose.layout.rememberColumnState
 import com.google.android.horologist.media.ui.navigation.MediaNavController.navigateToVolume
 import com.google.android.horologist.media.ui.navigation.MediaPlayerScaffold
 import com.google.android.horologist.media.ui.snackbar.SnackbarManager
 import com.google.android.horologist.media.ui.snackbar.SnackbarViewModel
 
 class MainActivity : ComponentActivity() {
+    private lateinit var navController: NavHostController
```

Comment:
Does `navController` need to be a member of `MainActivity`? Or can it be scoped inside `setContent`?

Reply:
switched back as it was.